TaskManager/Home/View/ViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ViewController.swift
+//  TaskManager
+//
+//  Created by user on 14.09.2022.
+//
+
+import UIKit
+
+enum Section: String, CaseIterable {
+    case Active
+    case Completed
+}
+
+protocol HomeView: AnyObject {
+    func presentModels(testData: [ModelTask])
+}
+
+class ViewController: UIViewController, UITableViewDelegate, HomeView {
+    
+    @IBOutlet weak private var editButton: UIButton!
+    @IBOutlet weak private var taskLabel: UILabel!
+    @IBOutlet weak private var tableView: UITableView!
+    @IBOutlet weak private var addButton: UIButton!
+    
+    private var arrayDiffable = [ModelTask]()
+    
+    private var dataSource: DataSourceDiffable!
+    var presenter: HomeViewPresenter!
+//    var delegate: DataStorage!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UINib(nibName: String(describing: TaskTableViewCell.self), bundle: nil), forCellReuseIdentifier: TaskTableViewCell.identifire)
+        configureTableView()
+        
+        addButton.setTitle("", for: .normal)
+        
+        presenter.viewDidLoad()
```

Comment:
call presenter.viewDidLoad() first so you have the data when you configure tableview